.github/workflows/pr-cli.yml

```diff
@@ -0,0 +1,39 @@
+name: PR check - CLI Installation test
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  cli-test:
+    runs-on: ${{ matrix.os }}
```

Comment:
the PR says "for different platforms" but it's only running on ubuntu-latest (IIUC the use of `matrix.os` from below)

Reply:
Yes, you are right, I will fix the description.